public/public.php

```diff
@@ -299,7 +302,7 @@ public function add_source_captions_to_content( $content ) {
 			}
 
 			// don’t show caption for own image if admin choose not to do so
-			if ( $this->is_standard_source( 'exclude' ) && self::use_standard_source( $id ) ) {
+			if ( Standard_Source::hide_standard_source_for_image( $id ) ) {
 				ISC_Log::log( sprintf( 'skipped "own" image for ID "%s"', $id ) );
 				continue;
```

Comment:
**Suggestion:** It seems that you are using the `Standard_Source` class methods statically. If `Standard_Source` is not designed as a utility class, consider instantiating it and using its methods as instance methods. This can make your code more testable and maintainable. [best practice]
```suggestion
   $standardSource = new Standard_Source();
   if ( $standardSource->hide_standard_source_for_image( $id ) ) {
       ISC_Log::log( sprintf( 'skipped "own" image for ID "%s"', $id ) );
       continue;
   }
```